blocked-edges/4.13.0-ec.1-NutanixCCMNeedsConfigMap.yaml

```diff
@@ -0,0 +1,13 @@
+to: 4.13.0-ec.1
+from: 4[.]12[.].*
+url: https://issues.redhat.com/browse/SPLAT-984
```

Comment:
I am not comfortable with linking the impact assessment jira card when the answers are not present. We can use the bug link here till the impact assessment card has more details.

Reply:
or we can copy over whatever the relevant bits from the OCPBUGS ticket are?  The impact-statement ticket is mutable, and anything we want to say, we can say there.

Reply:
It seems the current consensus is to wait for this one - if we decide we want to merge, we'll want to update SPLAT-984. I'm not doing that for now.